src/js/components/ErrorableCheckbox.jsx

```diff
@@ -33,7 +33,7 @@ class ErrorableCheckbox extends React.Component {
     // Calculate required.
     let requiredSpan = undefined;
     if (this.props.required) {
-      requiredSpan = <span className="form-required-span">*</span>;
+      requiredSpan = <span className="form-required-span">(*Required)</span>;
```

Comment:
I think you could update `CheckboxWidget.jsx` as well. You'll also need to run `npm run build` and commit the changes in the lib folder.

Reply:
@jbalboni Okay